src/runtime_src/hip/api/hip_memory.cpp

```diff
@@ -229,7 +215,7 @@ hipMalloc(void **ptr, size_t size)
 
 // Allocates device accessible host memory.
 hipError_t
-hipHostMalloc(void **ptr, size_t size, unsigned int flags)
+hipHostMalloc(void* *ptr, size_t size, unsigned int flags)
```

Comment:
It looks like all the changes about pointer typesetting are wrong.
```suggestion
hipHostMalloc(void** ptr, size_t size, unsigned int flags)
```

Reply:
fixed.